commands/codeOfConduct.js

```diff
@@ -0,0 +1,39 @@
+let ftl = require('../lib/ftl');
+
+module.exports.register = (botArguments) => {
+  const bot = botArguments.bot;
```

Comment:
You can be a bit terser about this if you want.

```suggestion
module.exports.register = ({ bot }) => {
```